TimberAPI/AssetLoader/AssetLoaderService.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace TimberbornAPI.AssetLoader
 {
-    public class AssetLoaderSystem : IAssetLoaderSystem
+    public class AssetLoaderService : IAssetLoaderService
```

Comment:
Not sure why *Service is better than *System. They are both very generic terms and personally I try to avoid them.